_pages/governance.md

```diff
@@ -230,6 +230,18 @@ Members of the BIDS Contributors Group are encouraged to support the
 BIDS specification by supporting the members of the Maintainers Group in
 their responsibilities.
 
+An *Active Contributor* is defined as a contributor that is reachable via
+the email address they provided to the BIDS maintainers team,
+and responds to correspondence by the BIDS group
+(for example, maintainers/steering group or the BIDS community at large).
```

Comment:
not sure of capitalization of "maintainers group" and "steering group". I'd just want it to be consistent with the rest of the document.